code/modules/client/preference_setup/loadout/loadout_head.dm

```diff
@@ -39,22 +35,13 @@
 	caps["grey cap"] = /obj/item/clothing/head/soft/grey
 	caps["white cap"] = /obj/item/clothing/head/soft/white
 	caps["flat cap"] = /obj/item/clothing/head/flatcap
-	caps["mailman cap"] = /obj/item/clothing/head/mailman
 	gear_tweaks += new/datum/gear_tweak/path(caps)
 
 /datum/gear/head/beret
-	display_name = "beret, red"
-	path = /obj/item/clothing/head/beret
-
-/datum/gear/head/beret/eng
 	display_name = "beret, engie-orange"
 	path = /obj/item/clothing/head/beret/engineering
 	allowed_roles = list("Station Engineer","Atmospheric Technician","Chief Engineer","Engineering Apprentice")
 
-/datum/gear/head/beret/purp
-	display_name = "beret, purple"
-	path = /obj/item/clothing/head/beret/purple
-
```

Comment:
Used by janitors.

Reply:
this is fine, since we have colorable berets and you can just make a purple beret. that way.